code/game/objects/items/devices/hivebot_crux.dm

```diff
@@ -0,0 +1,10 @@
+/obj/item/device/hivebot_crux
+	name = "hivebot crux"
+	desc = "A smooth, metallic metal cylinder with an indigo screen."
+	icon = 'icons/mob/hivebot.dmi'
+	icon_state = "hive_main-crash"
+	flags = CONDUCT | NOBLUDGEON
+	slot_flags = SLOT_BELT
+	materials = list(MAT_METAL = 500, MAT_GLASS = 1000)
+	origin_tech = "engineering=5;magnets=4;programming=4"
```

Comment:
The hivebot crux was originally intended to make a little helper companion that followed you around and would provide different stuff; i.e. if you were badly hurt, the little robot would detect that and stop any bleeding you might have and inject you with some very weak healing chemicals to tide you over. It's not intended for research only.